x-pack/plugins/ml/public/jobs/jobs_list/components/multi_job_actions/multi_job_actions.js

```diff
@@ -24,9 +24,10 @@ export class MultiJobActions extends Component {
 
   render() {
     const s = (this.props.selectedJobs.length > 1) ? 's' : '';
+    const jobsSelected = this.props.selectedJobs.length > 0;
```

Comment:
nit, can this condition be wrapped in brackets
`const jobsSelected = (this.props.selectedJobs.length > 0);`